### PR TITLE
89 acceptance tests

### LINKED ISSUE #89: Plan acceptance tests
## Issue Description
Plan the acceptance tests of the application to evaluate its usability, navigability and accessibility.  The points assessed must be based on the Nilsen Heuristics and the Accessibility Primer cited in the quality document.
The testers must fit the app’s personas.
Initial documentation must be made defining which flows will be tested, what is the minimum number of testers and what are the success criteria.

## Topics:
- [x] define the points to be evaluated
- [x] define the assessed flows
- [x] define success criteria
- [x] do the initial testing documentation

### PR DESCRIPTION
**Pull request by brendavsantos**
## Issue 
Resolves #89 

## Description
Creation and development of documentation regarding usability and acceptance

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

## How to validate?
Reading the document

### CODE REVIEW

File: docs/produto/acceptance-test.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* **Precisão**: Quantos erros a pessoa cometeu? E eles foram fatais ou a pessoa conseguiu se recuperar com as informações recebidas pelo sistema?              
+* **Desempenho**: Quanto tempo e quantos passos são necessários para que a pessoa complete tarefas básicas?       
+* **Lembrança**: O quanto a pessoa se lembra mais tarde ou depois de períodos sem usar?           
+* **Resposta Emocional**: Como a pessoa se sentiu depois de completar a tarefa? A pessoa estava confiante ou estressada?        
+
+O entendimento do usuário em relação às funcionalidades, à facilidade ou não em encontrar funcionalidades de acordo com a disposição na tela e a navegação, do login à execução das principais funcionalidades, serão as pontos principais de observação. Para padronizar esse processo, serão determinadas quais informações serão passadas ao usuário e quais os fluxos de teste.
```
Comment (herickport) [inline docs/produto/acceptance-test.md]: There is a minor error in "serão **as** pontos principais..." (line 11)

File: docs/produto/acceptance-test.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* **Resposta Emocional**: Como a pessoa se sentiu depois de completar a tarefa? A pessoa estava confiante ou estressada?        
+
+O entendimento do usuário em relação às funcionalidades, à facilidade ou não em encontrar funcionalidades de acordo com a disposição na tela e a navegação, do login à execução das principais funcionalidades, serão as pontos principais de observação. Para padronizar esse processo, serão determinadas quais informações serão passadas ao usuário e quais os fluxos de teste.         
+
+# Usuários de Teste
+Para que fosse possível o desenvolvimento da aplicação mesmo que sem um cliente real, foram criados perfis diversificados de usuários fictícios utilizando a técnica de [personas](https://github.com/fga-eps-mds/2020.1-stay-safe-docs/blob/master/docs/produto/product-definition.md). Para realizar os testes de aceitação é pretendido que a equipe encontre pessoas que possuam um perfil semelhante ao das personas, sendo estimada a participação de pelo menos cinco usuários e que o ambiente onde acontecerá os testes seja informal, havendo o mínimo de intervenções externas possíveis durante a realização dos mesmos.
```
Comment (herickport) [inline docs/produto/acceptance-test.md]: I would be better if the personas' document link was the one of the official documentation.

File: docs/produto/acceptance-test.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+Todos os testes deverão ser iniciados com a exposição do objetivo do produto, qual o contexto do desenvolvimento e qual o objetivo do teste que será realizado, enfatizando que o produto será avaliado e não a capacidade do usuário de realizar as tarefas. Além disso, é necessário:        
+* Como se dará o andamento do teste;       
+* Solicitar que o usuário expresse sempre o que está pensando e o que está tentando fazer;      
+* Relembrar o usuário de que o produto está em processo de aprimoramento, então não deve se preocupar em criticar pois sua crítica será de extrema importância para o resultado final.       
+
+## Realização de Tarefas
+* Tarefa 1: Será solicitado que o usuário realize seu cadastro na aplicação.      
+* Tarefa 2: Será solicitado que o usuário Cadastre uma ocorrência.      
+* Tarefa 3: Será solicitado que o usuário Visualize uma ocorrência.     
+* Tarefa 4: Será solicitado que o usuário visualize estatística do crime de roubo a veículo.
```
Comment (herickport) [inline docs/produto/acceptance-test.md]: I think that "estatística" should be in plural. (Task 4)

File: docs/produto/acceptance-test.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* Tarefa 5: Será solicitado que o usuário visualize estatísticas de uma cidade.
+* Tarefa 6: Será solicitado que o usuário avalie um bairro.         
+
+## Critérios de Sucesso   
+* O sucesso de cada tarefa é o usuário conseguir executá-la. 
+* Para cada uma das tarefas foi determinado um tempo mínimo, então caso usuário consiga executá-la dentro do tempo, demonstrará um bom padrão da usabilidade e reforçará o resultado positivo. 
+* Feedbacks dos testadores serem positivos(em sua maioria) também demonstrarão o sucesso dos testes.   
+
+### Heurísticas de Nielsen
+Conforme descrito no [documento de qualidade](https://fga-eps-mds.github.io/2020.1-stay-safe-docs/produto/quality/) do Stay Safe, os testes avaliarão se a aplicação atende as Heurísticas de Nielsen. Caso atenda, demonstra então que aplicação possui boa navegabilidade e usabilidade.
```
Comment (herickport) [inline docs/produto/acceptance-test.md]: You can omit the word "aplicação" or add an article "a" in "demonstra então que **aplicação** possui...". (line 62)

**Review by herickport — COMMENTED** [review]
The document is good, I suggested only some minor changes.

**Review by herickport — APPROVED** [review]

**Review by renan601 — APPROVED** [review]

**Pull request merged**